src/smcntrpmf.adoc

```diff
@@ -16,10 +16,10 @@ Smcntrpmf remedies these issues by introducing privilege mode filtering for the
 
 mcyclecfg and minstretcfg are 64-bit registers that configure privilege mode filtering for the cycle and instret counters, respectively.  
 
-[cols="^1,^1,^1,^1,^1,^1,^5",stripes=even,options="header"]
+[cols="^1,^1,^1,^1,^1,^1,^5,^1",stripes=even,options="header"]
 |====
-|63 |62 |61 |60 |59 |58 |57:0
-|0 |MINH |SINH |UINH |VSINH |VUINH |_WPRI_ 
+|63 |62 |61 |60 |59 |58 |57 |56
+|0 |MINH |SINH |UINH |VSINH |VUINH |_WPRI_ |_WPRI_
```

Comment:
If the resolution is just to make 57:56 WPRI, then I'm not sure this file needs any change.  57:0 were already WPRI, hence reserved.  @gfavor do you agree?  This PR leaves 55:0 undefined, which won't work.  Unlike mhpmeventX, those bits are not defined elsewhere.

Reply:
The Priv spec provides the default definition for all the bits of mhpmevent.  And note that the Priv spec does NOT specify WPRI for the bits of mhpmevent.  It specifies WARL.

The original ratified Sscofpmf extension said nothing about bits [55:0].  Which didn't leave them unspecified, but simply did not override the definition provided by the Priv spec.

Reply:
In a side email thread, it was agreed that smcntrpmf.adoc does not need any changes in this PR.  It already meets the requirement of keeping bits 57:56 WPRI and hence reserved.  So let's remove it from this changeset.

Reply:
Changes to Smcntrpmf have been removed.